websites/N/Nexon Logistics/metadata.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "Nexon Logistics",
+  "author": {
+    "id": "757284167972159489",
+    "name": "FaizurRahman#5151"
+  },
+  "description": {
+    "en": "Nexon Logistics is a virtual trucking company on TruckersMP."
+  },
+  "url": [
+    "nexonlogistics.com",
```

Site supports www subdomain
```suggestion
    "nexonlogistics.com",
    "www.nexonlogistics.com",
```